indexing-service/src/main/java/io/druid/indexing/common/task/IndexTask.java

```diff
@@ -151,7 +155,9 @@ public IndexTask(
       @JsonProperty("resource") TaskResource taskResource,
       @JsonProperty("spec") IndexIngestionSpec ingestionSchema,
       @JacksonInject ObjectMapper jsonMapper,
-      @JsonProperty("context") Map<String, Object> context
+      @JsonProperty("context") Map<String, Object> context,
+      @JacksonInject IndexMerger indexMerger,
+      @JacksonInject IndexIO indexIO
```

Comment:
the same concern also applies to all the other tasks, and classes that, until now, did not rely on jackson injection.